ipalib/__init__.py

```diff
@@ -942,12 +942,17 @@ def packages(self):
                 ipaserver.plugins,
             )
         else:
-            import ipaclient.remote_plugins
-            import ipaclient.plugins
-            result = (
-                ipaclient.remote_plugins.get_package(self),
-                ipaclient.plugins,
-            )
+            try:
+                import ipaclient.remote_plugins
+                import ipaclient.plugins
+                result = (
+                    ipaclient.remote_plugins.get_package(self),
+                    ipaclient.plugins,
+                )
+            except NetworkError:
+                #  instead of raising the default error connection message,
+                #  raising a more user-friendly one
+                raise GenericError('Cannot find IPA server to contact')
```

Comment:
This does not help anything at all. It actually makes the situation much worse. You are basically only omitting useful information for no reason. Please don't do that, you yourself can probably tell that it's not by a smallest bit clever.